pkg/osbuild/anaconda_stage.go

```diff
@@ -76,17 +63,15 @@ func filterEnabledModules(moduleStates map[string]bool) []string {
 }
 
 func NewAnacondaStageOptionsLegacy(enableModules, disableModules []string) *AnacondaStageOptions {
-	states := defaultModuleStates()
-	setModuleStates(states, enableModules, disableModules)
+	states := moduleStates(enableModules, disableModules)
 
 	return &AnacondaStageOptions{
 		KickstartModules: filterEnabledModules(states),
 	}
 }
 
 func NewAnacondaStageOptions(enableModules, disableModules []string) *AnacondaStageOptions {
-	states := defaultModuleStates()
-	setModuleStates(states, enableModules, disableModules)
+	states := moduleStates(enableModules, disableModules)
```

Comment:
Without `defaultModuleStates()`, isn't this whole function equivalent to
```go
func NewAnacondaStageOptions(enableModules []string) *AnacondaStageOptions {
	slices.Sort(enableModules)
	return &AnacondaStageOptions{
		ActivatableModules: slices.Compact(enableModules),
	}
}

```
?

Before, we needed to iterate through to apply user customizations to change the default states, but now it's all set outside the function already.

Reply:
Now that I think about it, it looks like this PR drops the ability for the blueprint to disable a module that's enabled by default.